ogr/services/pagure/service.py

```diff
@@ -139,8 +139,10 @@ def call_api(
             logger.error(response.json_content)
             if "error" in response.json_content:
                 error_msg = response.json_content["error"]
+                error_msg_ext = response.json_content["errors"]
                 raise PagureAPIException(
-                    f"Pagure API returned an error when calling `{url}`: {error_msg}",
+                    f"Pagure API returned an error when calling `{url}`: "
+                    f"{error_msg} - {error_msg_ext}",
```

Comment:
Can you add some test(s) please? e.g. wrong URL, wrong token, not-found object,...